.github/workflows/main.yml

```diff
@@ -82,3 +82,33 @@ jobs:
         else
           echo "No changes; skip commit"
         fi
+
+  json-spec:
+    name: Generate and upload json-spec
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: "1.15"
+    - name: Generate json-spec
+      # The docker image used in build-json-spec.sh has expected errors, but returns non-0
+      continue-on-error: true
```

Comment:
Vi ender ikke opp med å pushe ukomplett spec til bucket?

Reply:
Jepp, hang igjen fra en tidligere test for å få job-en til å kjøre gjennom. Takk :)